packages/govern-console/src/components/Header/Header.tsx

```diff
@@ -25,12 +25,12 @@ function Header() {
 
   const handleChangeChain = useCallback(
     e => {
-      setChainId(e.target.value)
+      updateChainId(Number(e.target.value))
       // When we change the chain ID, the DAO might not exist,
       // so we must revert back to the DAO selection screen.
       history.push(`/${getNetworkName(e.target.value)}`)
     },
-    [history, setChainId],
+    [history, updateChainId],
```

Comment:
I renamed `setChainId` to `updateChainId` to make it more clear that it is not the function from `useState()` but a custom one.

Reply:
Looks good!